config/default.yml

```diff
@@ -523,6 +523,16 @@ Style/MultilineOperationIndentation:
 Style/NumericLiterals:
   MinDigits: 5
 
+Style/OptionHash:
+  # Blacklist contains the default set of warnings to raise. If you do not
+  # want these to raise an error, you can override Blacklist.
+  Blacklist:
```

Comment:
That's not really a blacklist, so I'd prefer another name - e.g. OptionalParamNames. Let's hope you'll come up with something better.